src/styles/theme.js

```diff
@@ -77,9 +77,10 @@ export default ({ color, backgroundColor, borderColor }: Props) => ({
     alignItems: 'stretch',
   },
   link: {
-    marginTop: 10,
+    paddingTop: 10,
     fontSize: 15,
     color: BRAND_COLOR,
+    textAlign: 'right',
```

Comment:
Now whole row becomes clickable, then what is the purpose of aligning text to the right. Can we limit touchable area to the text itself.
![ezgif com-video-to-gif-2](https://user-images.githubusercontent.com/18511177/28151602-1c509ed6-67b9-11e7-9d81-8c307e88d0d1.gif)